content/docs/reference-events.md

```diff
@@ -244,33 +245,33 @@ string pointerType
 boolean isPrimary
 ```
 
-A note on cross-browser support:
+Una nota sobre la compatibilidad con varios navegadores:
 
-Pointer events are not yet supported in every browser (at the time of writing this article, supported browsers include: Chrome, Firefox, Edge, and Internet Explorer). React deliberately does not polyfill support for other browsers because a standard-conform polyfill would significantly increase the bundle size of `react-dom`.
+Los eventos de puntero aún no son compatibles con todos los navegadores (en el momento de escritura de este artículo, los navegadores compatibles incluyen: Chrome, Firefox, Edge e Internet Explorer). Reaccionar de forma deliberada no admite el polietileno para otros navegadores, ya que un polyfill de conformidad estándar aumentaría significativamente el tamaño del paquete de `react-dom`.
```

Comment:
The sentence:

> Reaccionar de forma deliberada no admite el polietileno para otros navegadores

Is not right, we're talking about the React library, not the verb, also, IMO `pollyfills` should not be translated.

Reply:
You got it! Makes complete sense! Resolving in next commit